openpype/hosts/maya/plugins/init/template_loader.py

```diff
@@ -0,0 +1,89 @@
+from openpype.plugins.load.abstract_load_template import AbstractTemplateLoader
```

Comment:
Plugins should be folder with python files(scripts) from which should not be imported.
This is changing it to module (with `__init__.py`  and imports) which is not right as it contain maya specific code. I suggest move content of file `./maya/api` (which is folder where maya specific code is) and imported from there.

Reply:
Solved